builders/manage/sudo/pause-transactions.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Pausing Transactions
+description: Learn how to use Sudo to temporarily pause hand-picked transactions, preventing their execution while allowing all other transactions to proceed as usual.
+---
+
+# Pausing Transactions
+
+## Introduction {: #introduction }
+
+The [Tx-pause module](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/tx-pause/src/lib.rs){target=\_blank} is one of the [built-in modules](/learn/framework/modules/#built-in-modules){target=\_blank} included in the Polkadot SDK, and it is available in any Tanssi appchain based on the [official templates](/builders/build/templates/overview/){target=\_blank} version [400](https://github.com/moondance-labs/tanssi/releases/tag/runtime-400-templates){target=\_blank} or higher.
+
+This module allows an appchain to avoid the execution of a set of hand-picked transactions momentarily while the rest of the transactions carry on as usual, accepting the calls. This feature is helpful in several scenarios, such as disabling functionality in which a security threat has been discovered, enabling seasonal functionality only when needed, enabling a set of transactions exactly on a launch date, and many others.
+
+In an emergency scenario, when a critical exploit is discovered, this module allows the appchain to isolate and stop only the affected functionality, effectively minimizing the overall impact. 
+
+!!! warning
+    At the time of this writing, this module hasn't yet been audited; therefore, it is not recommended for production use. 
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+For the examples in this guide, you will need to have the following:
+
+ - A Tanssi appchain (Snap or Dedicated)
```

Comment:
```suggestion
 - A Tanssi appchain (Snap or Dedicated) running runtime XXX or higher
```

We might need a dedicated section on how to check runtime version as it is popping up in multiple guides